automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/azure/AzureDiskSpec.scala

```diff
@@ -180,13 +201,52 @@ class AzureDiskSpec
           )
           _ = monitorCreateResult2.getStatus() shouldBe ClusterStatus.RUNNING
 
-          // TODO: https://broadworkbench.atlassian.net/browse/IA-4524, ssh into vm and verify disk contents
           disk2 <- getDisk2
           _ = disk2.getStatus() shouldBe DiskStatus.READY
           _ = disk2.getId() shouldBe monitorGetDisk.getId()
 
+          _ <- loggerIO.info("SSHing into second vm to verify disk contents")
+          output <- SSH.startBastionTunnel(RuntimeName(monitorCreateResult2.getRuntimeName())).use { t =>
+            for {
+              output <- SSH.makeSSHSession(t.hostName, t.port).use { session =>
+                SSH.executeCommand(session.makeSession, s"cat /home/jupyter/persistent_disk/test_disk.ipynb")
```

Comment:
So not getting an error message while trying to access the file means we are good, do I understand the intent correctly?

Reply:
On line 217 we verify that the contents of the file are what we put in there on the disk attached to the first runtime (which we verified on the first runtime on line 119)